packages/react/src/primitives/Collection/Collection.tsx

```diff
@@ -109,7 +111,15 @@ export const Collection = <Item,>({
         </Flex>
       ) : null}
 
-      {collection}
+      {items.length ? (
```

Comment:
Is there a possibility of items being undefined?

Reply:
Good questions. I don't think so because of line 67:
```
  items = Array.isArray(items) ? items : [];
 ```

Reply:
Wondering if it worth adding a check, just to be safe, in the future the code could be refactored and we may end up with NPE.